sig-compute/charter.md

```diff
@@ -0,0 +1,64 @@
+# SIG Compute Charter
+
+## Scope
+
+SIG-compute's scope is enormous, and includes node configurations, live-migration,
+virtualization, and much more.  
+
+In fact, currently sig-compute's scope captures the vast majority of the ares in the
+project, being the "default" sig for everything not directly related to the scope of other
+SIGs (such as storage and network).
+
+In the future we aim to break sig-compute into more sub-SIGs, which will be more granular
+and help define a concrete scope.
+
+### In scope
+
+Following topics are in-scope.
+
+We can use these topics as a base ground for thinking how to break sig-compute in the future.
+
+#### Node
+- SELinux
+- Cgroups
+- TPM
+- TSC
+- Swap
+- KSM
+- External kernel boot
+- seccomp
+- node-labeller
+- and more
+
+#### Virtualization
+- virt-launcher
+- libvirt / QEMU configuration
+- CPU features
+- Dedicated CPUs
+- Hypervisor features (e.g. HyperV)
+- and more
```

Comment:
I think you should also explicitly mention the devices management: generic host devices interface, GPU and vGPU discovery configuration.